### PR TITLE
IMP Ubicacions

### PR DESCRIPTION
**Pull request by tinogis**
### Description:

Adds administrative entities documentation and how to create a new village

### Tasks:

- [ ] Tasks to do
- [x] Tasks done

### Links:

> Add links for each file modified

* [Ubicacions](http://builds.gisce.net/powerp-docs/powerp_imp_other_country_villages/ca/base/ubicaciones/)
* [Ubicaciones](http://builds.gisce.net/powerp-docs/powerp_imp_other_country_villages/es/base/ubicaciones/)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_other_country_villages/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/base/ubicaciones.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+![](_static/ubicaciones/menu_ubicacion.png)
+
+### Divisions administratives bàsiques
+
+* **Estats**: S'identifiquen pel seu codi acrònim internacional de 2 lletres.
+Inclosos per defecte
+* **Comunitats autónomes**: Les comunitats autónomes d'Espanya s'identifiquen
+estan incloses per defecte. Per altres països es pot utilitzar per agrupacions
+de provínicies.
+* **Províncies**: Les províncies d'Espanya s'identifiquen per el seu codi INE i
```
Comment (jaumef) [inline docs/base/ubicaciones.md]: Tant el punt de **_províncies_** com el de **_municipis_** no s'estàn traduint correctament.
Revisa les claus al fitxer ".po"

**Comment by jaumef** [comment]
@tinogis fix conflicts and resolve revision changes

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_other_country_villages/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**